utility/ibm_dns_cleanup.py

```diff
@@ -0,0 +1,92 @@
+"""
+    Utility to cleanup orphan DNS record from IBM environment
+"""
+import sys
+from typing import Dict
+
+import yaml
+from docopt import docopt
+
+from compute.ibm_vpc import get_dns_service, get_dns_zone_id, get_ibm_service
+
+instance_id = "a55534f5-678d-452d-8cc6-e780941d8e31"
+
+doc = """
+Utility to cleanup orphan DNS record from IBM cloud.
+
+    Usage:
+        cleanup_ibm_dns_record.py --ibm-cred <cred-file>
```

Comment:
I like the thought of having `--ibm-cred`.

Would it make sense to leave it as `--creds`?